.github/CODEOWNERS

```diff
@@ -256,17 +256,9 @@
 
 # PRLabel: %Mgmt
 /sdk/batch/arm-batch/ @qiaozha @MaryGao
-<<<<<<< HEAD
-/sdk/batch/ci.mgmt.yml @qiaozha @MaryGao
 
 # PRLabel: %Mgmt
 /sdk/baremetalinfrastructure/arm-baremetalinfrastructure/ @qiaozha @MaryGao
-/sdk/baremetalinfrastructure/ci.mgmt.yml @qiaozha @MaryGao
```

Comment:
+1

Reply:
ci.mgmt.yml is moved to https://github.com/Azure/azure-sdk-for-js/blob/e964f85e642ea5299ec704ac3c6578e39908ee98/.github/CODEOWNERS#L1371